### DISCUSSION TITLE
Error Occurred After Building Addressables

### DISCUSSION THREAD

**Discussion by Robyiicorporate** [visualdesigncafe/nature-renderer#57] [Help]
I am writing to seek assistance regarding an error I encountered after building the addressables in Unity while using the Nature Renderer. Despite following the standard procedures, I am facing an issue that I am unable to resolve on my own.

**Error Details:**
  **Unity Version**: Unity 2021.3.20f1
  **Nature Renderer Version**: Nature renderer 2021.2.0
  **Error Messages:** 
  **1]**Shader error in 'NatureManufacture/URP/Foliage/Bark (Nature Renderer)': maximum ps_5_0 sampler register index (16) exceeded at line 382 (on d3d11).
  **2]**Shader error in 'NatureManufacture Shaders/Water/Water River Offset Vertex Color Flow': maximum ps_5_0 sampler register index (16) exceeded at line 538 (on d3d11).
  **3]**Shader error in 'NatureManufacture Shaders/Water/Water River Offset Vertex Color Flow': maximum ps_5_0 sampler register index (16) exceeded at line 538 (on d3d11).

I have attempted several troubleshooting steps, but the issue persists. I would greatly appreciate any guidance or suggestions you could provide to help resolve this issue.
If you need any additional information or files, please let me know, and I will be happy to provide them.
Thank you in advance for your support.

**Comment by maxplooi** [comment]
It looks like this is an issue with the **Nature Manufacture** shaders (not Nature Renderer), I recommended contacting them to resolve this issue.
